source/introduction/history-of-ethereum.rst

```diff
@@ -1,3 +1,70 @@
+
+.. https://blog.ethereum.org/2016/02/09/cut-and-try-building-a-dream/
+.. http://www.coindesk.com/ethererum-launches-cryptocurrency-2-0-network/
+
+
+The Ethereum Foundation and Crowdsale
+================================================================================
+In addition to developing the software for Ethereum, the ability to launch a new cryptocurrency and blockchain requires a massive bootstrapping effort in order to assemble the resources needed to get it up and running. To kickstart a large network of developers, miners, investors, and other stakeholders, Ethereum announced its plan to conduct a crowdsale of ether tokens, the currency unit of Ethereum. The legal and financial complexities of raising funds through a crowdsale led to the creation of several legal entities, including the Stiftung Ethereum (Ethereum Suisse), established in June 2014 in Zug, Switzerland. 
+
+Beginning in July 2014, Ethereum distributed the initial allocation of ether via a 42-day public crowdsale, netting 31,591 bitcoins, worth $18,439,086 at that time, in exchange for about 60,102,216 ether. The results of the sale were initially used to pay back mounting legal debts and also for the months of developer effort that had yet to be compensated, and to finance the ongoing development of the Ethereum. 
+
+* https://blog.ethereum.org/2014/07/22/launching-the-ether-sale/
+
+
+Ethereum Dev
+================================================================================
+Following the successful crowdsale, Ethereum development was formalized under a non-for-profit organization called Ethereum DEV (also known as ETH DEV), which manages the development of Ethereum under contract from Ethereum Suisse – with Vitalik Buterin, Gavin Wood, and Jeffrey Wilcke as the 3 directors of the organization. Developer interest in Ethereum grew steadily throughout 2014 and the ETH DEV team delivered a series of proof-of-concept (PoC) releases for the development community to evaluate. Frequent posts by ETH DEV team on the  `the Ethereum blog <https://blog.ethereum.org>`_ also kept the excitement and momentum around Ethereum going. 
+
+DEVcon-0
+--------------------------------------------------------------------------------
+In November 2014, ETH DEV organized the `DEVcon-0 event <https://blog.ethereum.org/2014/12/05/d%CE%BEvcon-0-recap/>`_, which brought together Ethereum developers from around the world to Berlin to meet and discuss a diverse range of Ethereum technology topics. Several of the presentations and sessions at DEVcon-0 would later drive important initiatives to make Ethereum more reliable, more secure, and more scalable. Overall, the event galvanized developers as they continued to work towards the launch of Ethereum. 
```

Comment:
I totally want that. Do you have the link? Like a table of contents page with title, description and deep link to specific cue point in the video for each